frontend/src/components/Histogram.jsx

```diff
@@ -1,7 +1,232 @@
 import React from "react";
+import { Container, Row, Col, Card } from "react-bootstrap";
+import {
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ComposedChart,
+  Bar,
+} from "recharts";
+
+const data = [
+  {
+    name: "2020-03-01",
```

Comment:
Can you rename these to eg. `date`, `cases_real` and `cases_counterfactual`?

Reply:
Done!